### ISSUE TITLE
Hangfire's LGPLv3 license violation report

### ISSUE THREAD

**Issue by odinserj** [dotnetcore/CAP#852]
Hello there, just want to let know to the larger audience that a lot of code in the `DotNetCore.CAP.Dashboard` project was simply copy-pasted from [Hangfire](https://www.hangfire.io/)'s [Dashboard UI](https://github.com/HangfireIO/Hangfire/tree/master/src/Hangfire.Core/Dashboard) without any permission or even notification. License was changed from LGPLv3 to MIT that's un-allowed by LGPLv3. Copyright notices were cleaned out and replaced with CAP ones that's un-allowed by **any** open-source license.

It's fine to copy ideas from open-source projects, source code when all the requirements are satisfied (including but not limited to copyright and notices) or designs, but not everything at the same time with wiping out previous copyright and license information. 

This is the directory structure that's cleanly matches Hangfire's one:

![image](https://user-images.githubusercontent.com/1078718/117003462-901d5480-aced-11eb-84bc-61755a18e056.png)

For example this is the [CommandDispatcher](https://github.com/dotnetcore/CAP/blob/master/src/DotNetCore.CAP.Dashboard/CommandDispatcher.cs) class from CAP:

![image](https://user-images.githubusercontent.com/1078718/117003497-98758f80-aced-11eb-9c6e-9d48ac9c87c9.png)

And this is the [CommandDispatcher](https://github.com/HangfireIO/Hangfire/blob/master/src/Hangfire.Core/Dashboard/CommandDispatcher.cs) class from Hangfire:

![image](https://user-images.githubusercontent.com/1078718/117003565-ae835000-aced-11eb-875f-972c15fd4671.png)

And even design looks the same as in [Hangfire](https://www.hangfire.io/img/ui/failed-jobs.png):

![image](https://user-images.githubusercontent.com/1078718/117004123-63b60800-acee-11eb-832b-0bfab1177239.png)

**Comment by yang-xiaodong** [comment]
Hello @odinserj , 
The early version of Dashboard and design of CAP comes from Hangfire UI, (this is because in the early days of .NET Core we did not find a suitable way to embed the page in the DLL), and we have not updated the Dashboard in the next few years. I did not know that LGPLv3 cannot be modified and used in MIT. We will use Blazor to completely replace our Dahboard in the next few weeks. I apologize for this and I'm very sorry for the impact on you or hangfire
